### ISSUE TITLE
Example project is not building on latest Defold version

### ISSUE THREAD

**Issue by subsoap** [defold/extension-spine#109]
```
/defold-spine/src/comp_spine_model.cpp
	Line 235: unused variable 'spine_model' [-Wunused-variable]
        SpineModelResource* spine_model = component->m_Resource;
                            ^
	Line 690: no member named 'GetRawObjects' in 'dmObjectPool<dmSpine::SpineModelComponent *>'
        dmArray<SpineModelComponent*>& components = world->m_Components.GetRawObjects();
                                                    ~~~~~~~~~~~~~~~~~~~ ^
	Line 749: no member named 'GetRawObjects' in 'dmObjectPool<dmSpine::SpineModelComponent *>'
        dmArray<SpineModelComponent*>& components = world->m_Components.GetRawObjects();
                                                    ~~~~~~~~~~~~~~~~~~~ ^
	Line 821: no member named 'GetRawObjects' in 'dmObjectPool<dmSpine::SpineModelComponent *>'
        dmArray<SpineModelComponent*>& components = world->m_Components.GetRawObjects();
                                                    ~~~~~~~~~~~~~~~~~~~ ^
	Line 890: no member named 'GetRawObjects' in 'dmObjectPool<dmSpine::SpineModelComponent *>'
        dmArray<SpineModelComponent*>& components = world->m_Components.GetRawObjects();
                                                    ~~~~~~~~~~~~~~~~~~~ ^
	Line 1271: unused variable 'constraint' [-Wunused-variable]
        spIkConstraint* constraint = component->m_SkeletonInstance->ikConstraints[*index];
                        ^
	Line 1297: unused variable 'constraint' [-Wunused-variable]
        spIkConstraint* constraint = component->m_SkeletonInstance->ikConstraints[*index];
                        ^
```

**Comment by subsoap** [comment]
![2022-10-28 13_02_49-About](https://user-images.githubusercontent.com/409170/198722515-794b3c03-8c45-4eb7-905f-be9a08245d8e.png)

**Comment by subsoap** [comment]
https://github.com/defold/extension-spine/releases/tag/2.4.0 this release works, just not master

**Comment by JCash** [comment]
Master works with our development version (i.e. 1.3.8+)

I guess we need to make some workflow changes to not confuse our users about this.
However, this issue won't be fixed, as it will resolve itself when the 1.3.8 will be released.

**Comment by subsoap** [comment]
@JCash need to make a new release? Updated to latest and now the latest release doesn't work but master does. :)
